ibis-server/app/model/coordinator.py

```diff
@@ -0,0 +1,33 @@
+from json import loads
+
+from app.mdl.rewriter import rewrite
+from app.model.data_source import DataSource, ConnectionInfo
+
+
+class Coordinator:
```

Comment:
```suggestion
class Connector:
```
How about calling it `Connector`? It's not a distributed architecture. `Coordinator` might not be suitable.

Reply:
Sure, nice name. Thanks.